server/container/guice/protocols/webadmin-mailbox/src/main/java/org/apache/james/modules/server/ReIndexingModule.java

```diff
@@ -50,5 +52,9 @@ protected void configure() {
         Multibinder.newSetBinder(binder(), TaskRegistration.class, Names.named(ONE_MAIL_TASKS))
             .addBinding()
             .to(MailboxesRoutes.ReIndexOneMailTaskRegistration.class);
+
+        Multibinder.newSetBinder(binder(), TaskRegistration.class, Names.named(ONE_MESSAGE_TASKS))
```

Comment:
This binding will not work for JPA that supports reindexing but not messageId.

We need to make this binding in a separated `MessageReIndexingModule` that can then be bound in cassandra/memory...